.github/workflows/test.yml

```diff
@@ -8,25 +8,24 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [12.x, 14.x]
```

Comment:
Haven't found docs on this, but it doesn't look like Yarn v2/v3 supports Node 10.

Seems okay for the design system, since all the apps we support are on Node 14.